idaes/models/flowsheets/demo_flowsheet.py

```diff
@@ -49,19 +51,137 @@ def build_flowsheet():
 
     TransformationFactory("network.expand_arcs").apply_to(m.fs)
 
+    m.fs.BT_props.set_default_scaling(
```

Comment:
For clarity, it might be helpful to break all of this out as a `set_scaling` method that is called by `main`.